### ISSUE TITLE
Pre-built binaries not installable for muhammara@5.0.0 and node@22.9.0 (node-v127 ABI, unknown)

### ISSUE THREAD

**Issue by unlight** [julianhille/MuhammaraJS#417]
Binaries not found, compile with node-gyp failed

```
info check checked for "\node_modules\muhammara\binding\muhammara.node" (not found)
http GET https://github.com/julianhille/MuhammaraJS/releases/download/5.0.0/node-v127-win32-x64-unknown.tar.gz
ERR! install response status 404 Not Found on https://github.com/julianhille/MuhammaraJS/releases/download/5.0.0/node-v127-win32-x64-unknown.tar.gz
WARN Pre-built binaries not installable for muhammara@5.0.0 and node@22.9.0 (node-v127 ABI, unknown) (falling back to source compile with node-gyp)
WARN Hit error response status 404 Not Found on https://github.com/julianhille/MuhammaraJS/releases/download/5.0.0/node-v127-win32-x64-unknown.tar.gz
ERR! UNCAUGHT EXCEPTION
ERR! stack Error: spawn EINVAL
ERR! stack     at ChildProcess.spawn (node:internal/child_process:420:11)
ERR! stack     at Object.spawn (node:child_process:761:9)
ERR! stack     at module.exports.run_gyp (\node_modules\@mapbox\lib\util\compile.js:80:18)
ERR! stack     at build (\node_modules\@mapbox\lib\build.js:41:13)
ERR! stack     at self.commands.<computed> [as build] (\node_modules\@mapbox\lib\js:86:37)
ERR! stack     at run (\node_modules\@mapbox\lib\main.js:81:30)
ERR! stack     at process.processTicksAndRejections (node:internal/process/task_queues:85:11)
ERR! System Windows_NT 10.0.19045
ERR! command "C:\\fnm_multishells\\node.exe" "\\node_modules\\@mapbox\\\bin\\ "install" "--fallback-to-build" "${EXTRA_NODE_PRE_GYP_FLAGS:-}```
```
Env: 
- node 22.9.0
- windows 10.0.19045

**Comment by julianhille** [comment]
true.
i'm sorry i somehow missed to add node 22 to the build. Only added it to musl builds.

**Comment by julianhille** [comment]
Has been released with 5.0.1
